lecture4/src/test/java/AddingOfProductToBasketTest.java

```diff
@@ -0,0 +1,30 @@
+import org.junit.Test;
+
+import static org.testng.Assert.assertTrue;
+
+public class AddingOfProductToBasketTest extends BaseTest {
+    @Test
+    public void signUp() {
+        driver.get("https://www.onliner.by");
+
+        OnlinerPage onlinerPage = new OnlinerPage(driver);
+        assertTrue(onlinerPage.isInitialized());
+
+        onlinerPage.showLoginForm();
+
+        LoginPage loginPage = new LoginPage(driver);
+        assertTrue(loginPage.isInitialized());
+
+        loginPage.fillData();
+        loginPage.clickToSubmitButton();
```

Comment:
ProductPage productPage = loginPage.clickToSubmitButton();
clickToSubmitButton() should return new ProductPage(driver);
public ProductPage clickToSubmitButton(){
...
return new ProductPage(driver);
}